scripts/cleanDB.sql

```diff
@@ -0,0 +1,23 @@
+delete from message_destinations_messages;
```

Comment:
This script is never referenced. Is it complete? How do we ensure it stays complete?